stories/core/Header.stories.tsx

```diff
@@ -278,3 +278,66 @@ export const CenterContent: Story<HeaderProps> = (args) => {
     </Header>
   );
 };
+
+export const SplitButtonHeader: Story<HeaderProps> = (args) => {
```

Comment:
Wonder if we should just modify existing stories, since split button really makes more sense in most cases.

Reply:
Modified stories to have mix of default and split buttons.